client/src/app/submission/submission.component.ts

```diff
@@ -88,11 +89,22 @@ export class SubmissionComponent implements OnInit {
           .subscribe(
             data => {
               this.submissionErrors = data['_embedded']['submissionErrors'];
-              const err = this.submissionErrors ? this.submissionErrors[0] : null;
-              if (err){
-                this.alertService.clear();
-                this.alertService.error(err['title'], err['detail'], false, false);
+              this.alertService.clear();
+              if (this.submissionErrors.length > this.MAX_ERRORS) {
+                this.alertService.error(
+                  `${ this.submissionErrors.length - this.MAX_ERRORS } Other Errors`,
+                  `Cannot show more than ${ this.MAX_ERRORS } errors, total errors: ${ this.submissionErrors.length }`,
+                  false,
+                  false);
+              }
+              let errors_displayed = 0;
+              for (const err of this.submissionErrors) {
+                if (errors_displayed >= this.MAX_ERRORS) {
+                  break;
+                }
                 console.log(err);
+                this.alertService.error(err['title'], err['detail'], false, false);
```

Comment:
This will create 9 error banners in the UI. What if we compose the error message first then add 1 error banner?

Reply:
We can also just give a "See all errors" link pointing to the Ingest Url which retrieves all the submission errors

Reply:
I looked into the top alert showing a link to the API for SubmissionErrors, but it's more complex than is sounds.